### PR TITLE
[MKC-1311] [MKC-1312] Add compatible hardware to Mouse & Keyboard

### PR DESCRIPTION
**Pull request by Hannes7eicher**
This PR adds a table of compatible hardware for the keyboard and mouse language reference.

### CODE REVIEW

File: Language/Functions/USB/Keyboard.adoc [hunk 1]
```diff
@@ -24,6 +24,31 @@ The library supports the use of modifier keys. Modifier keys change the behavior
 --
 // OVERVIEW SECTION ENDS
 
+[float]
+=== Compatible Hardware
+HID is supported on the following boards:
+[options="header"]
+|=========================================================
+| Board                | Pins
```
Comment (karlsoderby) [inline Language/Functions/USB/Keyboard.adoc:32]: Looks good. I think, let's not link all MKR boards, but instead link to the family, like this: `https://docs.arduino.cc/#mkr-family`.

Comment (karlsoderby) [inline Language/Functions/USB/Keyboard.adoc:32]:
```suggestion
| Board                | Supported Pins                     
```

File: Language/Functions/USB/Keyboard.adoc [hunk 2]
```diff
@@ -24,6 +24,31 @@ The library supports the use of modifier keys. Modifier keys change the behavior
...
 // OVERVIEW SECTION ENDS
 
+[float]
+=== Compatible Hardware
+HID is supported on the following boards:
+[options="header"]
+|=========================================================
+| Board                | Pins                     
+| link:https://docs.arduino.cc/hardware/leonardo[Leonardo]             | Digital / Analog Pins 
+| link:https://docs.arduino.cc/hardware/micro[Micro]                   | Digital / Analog Pins
```
Comment (karlsoderby) [inline Language/Functions/USB/Keyboard.adoc:34]: Let's change `Digital / Analog Pins` to `All digital & analog pins

**Review by karlsoderby — CHANGES_REQUESTED** [review]
Looks good! Couple of suggestions before we can merge!

**Review by karlsoderby — APPROVED** [review]

**Comment by per1234** [comment]
@Hannes7eicher @karlsoderby really??????

What makes you think pins are relevant to the subject of HID?

What sort of verification did you do for the list of boards? These libraries do not support the Nano ESP32 and GIGA R1 WiFi, which you would have learned by simply attempting to compile any sketch that uses either library for these boards.

**Pull request merged**